### PR TITLE
Signal-Desktop: update to 1.14.1

### PR DESCRIPTION
**Pull request by julio641742**

### CODE REVIEW

**Comment by cr6git** [comment]
I'm sure @pullmoll would like to review this. :thinking: https://github.com/void-linux/void-packages/commit/d9576f500e4704a3eb4935e20e717080a3c6ddb4

**Comment by maxice8** [comment]
@cr6git then request his review

**Comment by pullmoll** [comment]
Why would I like to review this? :stuck_out_tongue_closed_eyes: I'm clueless and no Signal user.

File: srcpkgs/Signal-Desktop/template [hunk 1]
```diff
@@ -39,8 +40,7 @@ do_install() {
...
 
 	vmkdir usr/share/icons/hicolor
-	for size in 16 24 32 48 64 128 256 512 1024; do
-		vinstall build/icons/png/${size}x${size}.png 644 usr/share/icons/hicolor/${size}x${size}/apps/	signal.png
+	for size in 16 32 48 128 256 1024; do
+		vinstall images/icon_${size}.png 644 usr/share/icons/hicolor/${size}x${size}/apps/	signal.png
 	done
-	npm remove -g yarn
-}
+}
```
Comment (pullmoll) [inline srcpkgs/Signal-Desktop/template]: Line break missing.

**Comment by julio641742** [comment]
ready for another review

File: srcpkgs/Signal-Desktop/template [hunk 2]
```diff
@@ -28,7 +26,6 @@ do_install() {
 	case "$XBPS_TARGET_MACHINE" in
 		i686*) _ARCH="-ia32-";;
 		x86_64*) _ARCH="-";;
-		*) _ARCH="-${XBPS_TARGET_MACHINE%-musl}-";;
```
Comment (Vaelatern) [inline srcpkgs/Signal-Desktop/template]: You have to explain the removal of this line please

File: srcpkgs/Signal-Desktop/template [hunk 3]
```diff
@@ -39,8 +36,8 @@ do_install() {
...
 
 	vmkdir usr/share/icons/hicolor
-	for size in 16 24 32 48 64 128 256 512 1024; do
-		vinstall build/icons/png/${size}x${size}.png 644 usr/share/icons/hicolor/${size}x${size}/apps/	signal.png
+	for size in 16 32 48 128 256 1024; do
+		vinstall images/icon_${size}.png 644 usr/share/icons/hicolor/${size}x${size}/apps/	signal.png
 	done
-	npm remove -g yarn
 }
+
```
Comment (Vaelatern) [inline srcpkgs/Signal-Desktop/template]: No need for this newline

**Pull request merged**